apps/ledger-live-desktop/src/storyly/useStoryly.tsx

```diff
@@ -0,0 +1,73 @@
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { StorylyInstanceID } from "@ledgerhq/types-live";
+import { useLayoutEffect, useRef } from "react";
+import { useSelector } from "react-redux";
+import { Language } from "~/config/languages";
+import { languageSelector } from "~/renderer/reducers/settings";
+import { StorylyStyleProps, useStorylyDefaultStyleProps } from "./style";
+
+/**
+ * Storyly Options
+ */
+type StorylyOptions = {
```

Comment:
I think that we need to agree with the team on how we want to comment/document our component. IMO this is the kind of comments we don't need and just generate noise.

Reply:
You are right, to some extent that could create unnecessary noise. I'll leave it as is for now but yeah let's agree with the team on how/when/what to comment/document.